### ISSUE TITLE
Crash when trying to post after iPhone upgrade

### ISSUE THREAD

**Issue by defjaf** [microdotblog/issues#129]
I just upgraded to a new iPhone (from 7 to XS, although I don't think that matters) -- restored from a backup of the old phone. Now, when I try to post from the iOS client, the app crashes -- and loses my in-progress post! I can still post from the Mac and iPad clients, so I'm pretty sure it's a client config issue -- something gets lost or misconfigured in the restore from backup. 

I have also reported this to [@help](http://micro.blog/help) as [@jaffe](http://micro.blog/jaffe).

I understand that Micro.blog User [@smokey](http://micro.blog/smokey) has also seen similar behaviour).

**Comment by manton** [comment]
You're right, this can happen after a phone upgrade. Best work-around until we fix it is to sign out of the Micro.blog app and back in, which reset the settings. Thanks!
